webserver/templates/macros.html

```diff
@@ -13,38 +13,37 @@
 {% endmacro %}
 
 {% macro pager(links, prevpage, current_page, last_page, nextpage) %}
-	<nav>
-	  <ul class="pagination">
-		{% if current_page == 1 %}
-		<li class="disabled">
-        <span aria-hidden="true">&laquo;</span>
-		{% else %}
-		<li>
-          <a href="{{prevpage}}" aria-label="Previous">
-			<span aria-hidden="true">&laquo;</span>
-		  </a>
-		{% endif %}
-		</li>
-		{% for number, link in links %}
-			{% if current_page == number %}
-			<li class="active">
-			{% else %}
+    <nav>
+        <ul class="pagination">
+            {% if current_page == 1 %}
+            <li class="disabled">
+                <span aria-hidden="true">&laquo;</span>
+            {% else %}
             <li>
+                <a href="{{prevpage}}" aria-label="Previous">
+                    <span aria-hidden="true">&laquo;</span>
+                </a>
+            {% endif %}
+            </li>
+            {% for number, link in links %}
+                {% if current_page == number %}
+                <li class="active">
+                {% else %}
+                <li>
```

Comment:
You can reduce these to

    <li {{ class=active if current_page == number }}>